tests/config/bindings/azure/storagequeues/bindings.yml

```diff
@@ -12,6 +12,6 @@ spec:
   - name: storageAccessKey
     value: ${{AzureBlobStorageAccessKey}}
   - name: queue
-    value: ${{AzureBlobStorageQueue}}
+    value: conf-tests
```

Comment:
Is this an intentional change? I think it's technically isn't a secret so probably fine hard-coded, but just making sure that this isn't a leftover line to running tests locally.

Reply:
Yeah that was a miss-click. service queue vs storage queue, changed it back.